docs/UsersGuide.asciidoc

```diff
@@ -989,9 +995,14 @@ in the operators menu and select a group. The size limit for assets can be
 configured under 'Edit job group properties'. It also shows the size of
 assets which belong to that group and not to any other group.
 
-Job groups inherit the size limit from their parent group unless the limit
-is set explicitely. The default size limit for groups can be adjusted in
-the +default_group_limits+ section of the openQA config file.
+If a job group is part of a parent group the size limit is set on parent
+group level and the limit set on job group level is ignored. So even if the
```

Comment:
this should then also be removed - as there is no limit on job groups in parents worth it.